shardingsphere-infra/shardingsphere-infra-binder/pom.xml

```diff
@@ -43,5 +43,12 @@
             <artifactId>shardingsphere-infra-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-sql-parser-mysql</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
The infra module can not understand concrete module such as shardingsphere-sql-parser-mysql